README.md

```diff
@@ -74,13 +74,13 @@ What will the following return?
 
 ```js
 typeof 4
-
+// number
 typeof "hello"
-
+// 'String"
 typeof true
-
+// Bolean
 2 === 1 || 3 === 4
-
+// Bolean, false
```

Comment:
Remember types are lowercase when using typeof. Great job on everything else though!

Reply:
Appreciated! Thank you